src/jolt/vm/test_vm.rs

```diff
@@ -148,4 +166,56 @@ mod tests {
       "Unused enum variants in Subtables"
     );
   }
+
+  #[test]
+  fn subtable_reuse() {
+    // Setup VM / Instructions / Subtables
+    #[repr(u8)]
+    #[derive(Copy, Clone, EnumIter, EnumCountMacro)]
+    #[enum_dispatch(JoltInstruction)]
+    pub enum ReuseInstructionSet {
+      BNE(BNEInstruction),
+      BGEU(BGEUInstruction),
+    }
+    impl Opcode for ReuseInstructionSet {}
```

Comment:
why not use the macro for this?

Reply:
It was running into reimport issues. We have to sort out the macro export at some point if we want to keep using the macro. IIRC this is difficult due to the non-idempotence of enum_dispatch.